facia-tool/app/auth/PanDomainAuthActions.scala

```diff
@@ -42,8 +45,8 @@ trait PanDomainAuthActions extends AuthActions with Results {
 
   override lazy val domain: String = Configuration.faciatool.pandomainDomain
   override lazy val system: String = Configuration.faciatool.pandomainService
+  override def awsCredentialsProvider  = new AWSCredentialsProviderChain(
+    new STSAssumeRoleSessionCredentialsProvider(Configuration.faciatool.pandomainRoleArn, "workflow")
```

Comment:
Is it worth leaving the secret in the chain for applications that don't live in AWS (which of course includes local development)?